pkg/apis/v1beta1/nodepool.go

```diff
@@ -135,6 +137,50 @@ func (in *NodePool) Hash() string {
 	})))
 }
 
+const qualifiedNameMaxLength int = 63
+const DNS1123SubdomainMaxLength int = 253
+
+func (in *NodePool) ValidateQualifiedName() (errs error) {
+	for _, taint := range in.Spec.Template.Spec.Taints {
```

Comment:
Just curious: Is there an upstream function that does this that we could leverage?